frontend/src/console/components/ConsolePage/__snapshots__/ConsolePage.spec.tsx.snap

```diff
@@ -307,41 +307,10 @@ exports[`Console Page Correctly renders page with data 1`] = `
     </h2>
   </div>
   <div
-    className="d-flex justify-content-center align-middle redeployment-toolbar"
+    className="fade text-center alert alert-warning show"
+    role="alert"
   >
-    <div
-      className="redeployment-nav"
-    >
-      <span
-        className="status-pill badge badge-pill badge-primary"
-        onBlur={[Function]}
-        onFocus={[Function]}
-        onMouseOut={[Function]}
-        onMouseOver={[Function]}
-      >
-        <span
-          className="dot"
-          style={
-            Object {
-              "backgroundColor": "#ff9b04",
-            }
-          }
-        />
-        <div
-          className="text"
-        >
-          Status: We're preparing your instance 
-        </div>
-      </span>
-      <button
-        className="float-right loading btn btn-primary btn-lg"
-        disabled={true}
-        onClick={[Function]}
-        type="button"
-      >
-        Deploy (0 updates)
-      </button>
-    </div>
+    Some of your email addresses are pending confirmation. Please confirm them to begin managing your Open edX instance.
```

Comment:
@xitij2000, @rusrushal13, there is just one email address for the user to confirm here, right? If so, can this be updated to `Please confirm your email address to begin managing your Open edX instance.`?

CC @giovannicimolin